nasa/earthdata-search
# EDSC-4152: Converted granule_details from cypress to playwright

# Overview

### What is the feature?

Converted the granule_details cypress test to playwright.

### What is the Solution?

- Moved the file to be under the playwright e2e testing path (mocks and actual testing file)
- Created the .spec playwright testfile and converted all the cypress testing syntax to playwright.

### What areas of the application does this impact?

Just the playwright and cypress tests.

# Testing

### Reproduction steps

- **Environment for testing: local**
- **Collection to test with: N/A**

1. run ```npm run playwright:ui```
2. run the "paths" tests or expand the "paths" tests till you get to the granule_details tests and run the test.
2. observe the tests passing

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
